Makefile

```diff
@@ -78,6 +80,11 @@ frontend_tests_headless: frontend_deps
 frontend: frontend_deps
 	for i in pkg $(WWW_PKGS); do $(PIP) install -e $$i || exit 1; done
 
+# build frontend wheels for installation elsewhere
+frontend_wheels: frontend_deps
```

Comment:
I didn't find any good testing documentation for the frontend so I used this and deployed the buildbot_www package on my buildbot master.